data/reference-cards.js

```diff
@@ -1,160 +1,176 @@
-    "title": "Conditions",
-    "subtitle": "Reference Card",
-    "image": "reference-cards/Conditions.png"
-  },
-  {
-    "id": 23,
-    "title": "Reinforce Action",
-    "subtitle": "Reference Card",
-    "image": "reference-cards/ReinforceAction.png"
-  },
-  {
-    "id": 24,
-    "title": "Bomb Tokens",
-    "subtitle": "Reference Card #08",
-    "image": "reference-cards/BombTokens-08.png"
-  },
-  {
-    "id": 25,
-    "title": "Bullseye Firing Arc",
-    "subtitle": "Reference Card",
-    "image": "reference-cards/BullseyeFiringArc.png"
-  },
-  {
-    "id": 26,
-    "title": "Reload Action and Jam Tokens",
-    "subtitle": "Reference Card",
-    "image": "reference-cards/ReloadActionAndJamTokens.png"
+   {
+      "id": 0,
+      "title": "Ion Token",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/IonToken.png"
+  },
+   {
+      "id": 1,
+      "title": "Bomb Tokens",
+      "subtitle": "Reference Card #01",
+      "image": "reference-cards/BombTokens.png"
+  },
+   {
+      "id": 2,
+      "title": "Boost Action",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/BoostAction.png"
+  },
+   {
+      "id": 3,
+      "title": "Modifications and Titles",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/ModificationsAndTitles.png"
+  },
+   {
+      "id": 4,
+      "title": "Using Bombs",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/UsingBombs.png"
+  },
+   {
+      "id": 5,
+      "title": "Bomb Tokens",
+      "subtitle": "Reference Card #02",
+      "image": "reference-cards/BombTokens-02.png"
+  },
+   {
+      "id": 6,
+      "title": "Cloak Action",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/CloakAction.png"
+  },
+   {
+      "id": 7,
+      "title": "Decloak",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/Decloak.png"
+  },
+   {
+      "id": 8,
+      "title": "Segnor’s Loop",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/SegnorsLoop.png"
+  },
+   {
+      "id": 9,
+      "title": "Bomb Tokens",
+      "subtitle": "Reference Card #04",
+      "image": "reference-cards/BombTokens-04.png"
+  },
+   {
+      "id": 10,
+      "title": "Using Cluster Mines",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/UsingClusterMines.png"
+  },
+   {
+      "id": 11,
+      "title": "SLAM Action",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/SlamAction.png"
+  },
+   {
+      "id": 12,
+      "title": "Turret Primary Weapon",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/TurretPrimaryWeapon.png"
+  },
+   {
+      "id": 13,
+      "title": "Bomb Tokens",
+      "subtitle": "Reference Card #03",
+      "image": "reference-cards/BombTokens-03.png"
+  },
+   {
+      "id": 14,
+      "title": "Tractor Beam Token",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/TractorBeanToken.png"
```

Comment:
This needs to be changed to `reference-cards/TractorBeamToken.png` as you renamed the image.

Reply:
This is why the build is failing.